src/FirebaseArray.js

```diff
@@ -314,7 +314,7 @@
             }
             rec.$id = $firebaseUtils.getKey(snap);
             rec.$priority = snap.getPriority();
-            $firebaseUtils.applyDefaults(rec, this.$$defaults);
+            angular.extend(rec, this.$$defaults);
```

Comment:
Oops, @katowulf is right on this one. This means `applyDefaults` needs to stay - it is slightly (but importantly) different than `angular.extend`. You could do `angular.extend(rec, this.$$defaults, rec)` but that's not as efficient as what we have, and a little confusing.